core/backend/src/rpc-impl/IModelTileRpcImpl.ts

```diff
@@ -223,6 +232,7 @@ export class IModelTileRpcImpl extends RpcInterface implements IModelTileRpcInte
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     return IModelHost.tileCacheService!.obtainContainerUrl(id, expiry, clientIp);
   }
+  /* eslint-enable deprecation/deprecation */
 
   public async isUsingExternalTileCache(): Promise<boolean> { // eslint-disable-line @itwin/prefer-get
     return IModelHost.usingExternalTileCache;
```

Comment:
this only checks `tileStorage` but not `tileCacheService`. Let's talk tomorrow about how we can handle this better.